testsuite/testsuite_test.go

```diff
@@ -6,15 +6,107 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
 	"math/rand"
 	"net/http"
 	"strings"
 	"testing"
 	"time"
 
 	"gotest.tools/v3/assert"
+
+	"github.com/ethereum/go-ethereum/crypto"
 )
 
+func TestTransactionAndCQLAndRead(t *testing.T) {
+
+	//Test persona
+	privateKey, err := crypto.GenerateKey()
+	assert.NilError(t, err)
+	signerAddr := crypto.PubkeyToAddress(privateKey.PublicKey).Hex()
+	username, deviceID, personaTag := triple(randomString())
+	c := newClient(t)
+	assert.NilError(t, c.registerDevice(username, deviceID))
+
+	resp, err := c.rpc("nakama/claim-persona", map[string]any{
+		"persona_tag":    personaTag,
+		"signer_address": signerAddr,
+	})
+	assert.NilError(t, err, "claim-persona failed")
+	assert.Equal(t, 200, resp.StatusCode, copyBody(resp))
+
+	assert.NilError(t, waitForAcceptedPersonaTag(c))
+	type CreatePlayerTxMsg struct{ Nickname string }
```

Comment:
i don't think you need to define this, you can just import it from the template - `"github.com/argus-labs/starter-game-template/cardinal/tx"`

i have a suspicion that this might complicate the build/testing process though... if it does u can just resolve and leave as is.

Reply:
yeah this caused problems.